README.md

```diff
@@ -5,24 +5,23 @@ Loads in less than 10kb (excluding favicons) without any client side javascript.
 ## Run wxkb.io locally
 1. [Install node.js](https://nodejs.org/en/download/)
   * if you have [homebrew](http://brew.sh/), `brew install node`
-1. clone repo, `git clone https://github.com/JulianNorton/weather-10kb.git`
-1. `npm install`
```

Comment:
Markdown converts the `.1` to numbered lists automatically, no need to edit this!